test/integration/base.py

```diff
@@ -243,7 +243,7 @@ def snowflake_profile(self):
         }
 
     def bigquery_profile(self):
-        credentials_json_str = os.getenv('BIGQUERY_SERVICE_ACCOUNT_JSON').replace("'", '')
+        credentials_json_str = os.getenv('BIGQUERY_TEST_SERVICE_ACCOUNT_JSON').replace("'", '')
```

Comment:
I know this isn't code you added, but why is this replace necessary? might be worth a comment if we know.